### PR TITLE
Add Notice message for more than on property in same statement definition

### LINKED ISSUE #165: [Analyzer] One Property definition per line
It notices on: `public $a, $b=1;` or something like that.
So we need a statement analyzer for Property Node.
It's very easy, so it's good for someone who is new to the project.

How to do it:
1. Create Analyzer (you can use this one as a template: https://github.com/ovr/phpsa/blob/master/src/Analyzer/Pass/Statement/DoNotUseGoto.php )
and register it to Stmt\Property. Now:
Check that `$stmt->props` (array) contains more than one entry. If yes write a nice notice message, else return false.
2. Add it to PHPSA\Analyzer\Factory (registerStatementPasses)
3. Write Tests (analyze-fixtures directory): One example with one property and one with 2 or more (again look at the existing ones for statements)

Check our Contribution Guide and create a pull request.
Thanks :cake:

### PR DESCRIPTION
**Pull request by barroca**
Hey!

Type: new feature

Link to issue:
Closes #165 

This pull request affects the following components: **(please check boxes)**

* [ ] Core
* [x] Analyzer
* [ ] Compiler
* [ ] Control Flow Graph
* [ ] Documentation

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines](/.github/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I wrote some tests for this PR.

Small description of change:

-Add Notice Message when definition of multiple properties on the same statement.
-Fix typo of division word

Thanks

### CODE REVIEW

File: tests/analyze-fixtures/Statement/TestMultipleProperties.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+?>
+----------------------------
+[
+    {
+        "type":"limit.properties",
+        "message":"Number of properties larger than one.",
+        "file":"TestMultipleProperties.php",
+        "line":6
```
Comment (ddmler) [inline tests/analyze-fixtures/Statement/TestMultipleProperties.php]: needs to be 8 instead of 6 (always the line before the thing it notices on)

Comment (barroca) [inline tests/analyze-fixtures/Statement/TestMultipleProperties.php]: done

**Comment by barroca** [comment]
@ddmler I think the fixture has some error it is returning a fatal:
```Notice:  Number of properties larger than one. in ./fixtures/Compiling/statements/MultiplePropertiesDefinition.php on 13 [limit.properties]

fatal: no such path 'fixtures/Compiling/statements/MultiplePropertiesDefinition.php' in HEAD```

**Comment by ddmler** [comment]
Thanks @barroca travis says the fixture is good too. :+1:

**Pull request merged**